frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditCheckbox.tsx

```diff
@@ -238,11 +239,17 @@ export const EditCheckbox = (props: EditCheckboxProps): JSX.Element => {
               name="ValidationOptions.customMin"
               control={control}
               rules={customMinValidationOptions}
-              render={({ field }) => (
+              render={({ field: { onChange, ...rest } }) => (
                 <NumberInput
+                  inputMode="numeric"
+                  precision={0}
                   flex={1}
                   showSteppers={false}
-                  {...field}
+                  onChange={(val) => {
+                    // Only allow numeric inputs
+                    onChange(val.replace(/\D/g, ''))
```

Comment:
should we transform output of NumberInput to be `number` type directly? This better fits with the `Number` type in the schema.

Reply:
[6744778](https://github.com/opengovsg/FormSG/pull/3702/commits/6744778bc42caf15ca8ddd6ffcf75568869db583)